userexperior-cordova-plugin/src/ios/CDVUserExperior.m

```diff
@@ -60,8 +60,16 @@ - (void)pauseRecording:(CDVInvokedUrlCommand*)command {
 }
 
 - (void)stopRecording:(CDVInvokedUrlCommand*)command {
-    [UserExperior stopRecording];
+    if (command.arguments.count == 0) {
+        // No arguments provided, call the function without resetAppSessionID
+        [UserExperior stopRecording];
+    } else {
+        // If any argument (boolean) provided, call the function with resetAppSessionID
+        BOOL resetAppSessionID = [[command.arguments objectAtIndex:0] boolValue];
+        [UserExperior stopRecordingWithResetSession:resetAppSessionID];
```

Comment:
Is this correct function name, as I couldn't find this function in the native SDK here:

[UserExperior-Development/UserExperiorSDK/UserExperiorSDK/public/UserExperior.swift](https://github.com/devrev/ue-ios-sdk/pull/1/files#diff-4cd87540b339727525a7d2a4dcaf2ce827ef51f4deda25080dc0d9e5e9881bef)

Reply:
this was slightly different in obj c. I updated it to the correct one.